tools/sdk/solend/refreshReserve.ts

```diff
@@ -0,0 +1,26 @@
+import { TransactionInstruction } from '@solana/web3.js'
+import { refreshReserveInstruction } from '@solendprotocol/solend-sdk'
+import {
+  SolendDeposableAndWithdrawableSupportedMint as SolendDeposableAndWithdrawableSupportedMint,
+  SOLEND_ADDRESSES_PER_TOKEN,
+  SOLEND_PROGRAM_ID,
+} from './constant'
+
+export async function refreshReserve({
+  mintName,
+}: {
+  mintName: SolendDeposableAndWithdrawableSupportedMint
+}): Promise<TransactionInstruction> {
```

Comment:
Here might be able to return an array in some case

Reply:
Do not apply due to the fact the mintName is specified for now, thus only TransactionInstruction is returned